src/k8s/pkg/component/component.go

```diff
@@ -16,6 +16,8 @@ import (
 type ComponentManager interface {
 	// Enable enables a k8s component.
 	Enable(name string) error
+	// EnableWithValues enables a k8s component with custom configuration.
+	EnableWithValues(name string, values map[string]any) error
```

Comment:
We fully control callers here, so let's just change `Enable`?

```suggestion
	// Enable enables a k8s component, optionally specifying custom configuration options.
	Enable(name string, values map[string]any) error
```